sway-lib-std/src/token.sw

```diff
@@ -95,13 +97,23 @@ pub fn transfer_to_output(amount: u64, asset_id: ContractId, to: Address) {
     // If an output of type `OutputVariable` is found, check if its `amount` is
     // zero. As one cannot transfer zero coins to an output without a panic, a
     // variable output with a value of zero is by definition unused.
-    let outputs_count = tx_outputs_count();
-    while index < outputs_count {
-        if tx_output_type(index) == OUTPUT_VARIABLE && tx_output_amount(index) == 0 {
-            output_index = index;
-            output_found = true;
-            break; // break early and use the output we found
-        };
+    let outputs = output_count();
+    //log(outputs);
+    while index < outputs {
+        let type_of_output = output_type(index);
+        log(type_of_output);
```

Comment:
Does this function, from the `std`, need to log?

Reply:
Oh shoot no. I'll remove. Good catch.

Reply:
fixed.